TestApp/app/src/main/java/com/example/testapp/ui/BaseActivity.kt

```diff
@@ -0,0 +1,6 @@
+package com.example.testapp.ui
+
```

Comment:
Move all the Base level files to a package called as base

Reply:
cleared base level classes considering references and requirement